app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:weightSum="1"
+    android:background="@drawable/my_recycleview_element_border"
+    android:padding="@dimen/album_art_border">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/album_art_size"
+        android:orientation="vertical"
+        android:background="@drawable/my_recycleview_element_border"
+        android:padding="@dimen/album_info_border">
+
+        <TextView
+            android:id="@+id/album_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:text="36 Chambers"
```

Comment:
never hard code text into a layout. If you want to see in in the design window then use tools:text="36 Chambers" - this means it won't get compiled into the APK and so could never appear in the app by accident. you should also put 'tools' items last within each view's xml